components/BottomSheet/index.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React, {useState, useCallback} from 'react';
+import {View, Text, TouchableOpacity, Dimensions} from 'react-native';
+import styles from './style';
+
+const BottomSheet = ({children, onClose}) => {
+  const [sheetHeight, setSheetHeight] = useState(Dimensions.get('window').height * 2);
+
+  const handleClose = useCallback(() => {
+    setSheetHeight(0);
+    onClose();
+    }, [onClose]);
+
+  return (
+    <TouchableOpacity
```

Comment:
Why use TouchableOpacity

Reply:
it is unnecessary here, i will change it.